### PR TITLE
adds L for opening up leaderboard

### PR DESCRIPTION
**Pull request by SaraVieira**
Opens small window on the bottom like help with the best scores

CleanShot 2021-06-10 at 18 27 13@2x

### CODE REVIEW

File: src/utils/data/leaderboard.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { supabase } from '../initSupabase'
```
Comment (njm222) [inline src/utils/data/leaderboard.js:1]: Exactly what I wanted thanks!

File: src/ui/LeaderBoard.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+          </button>
+          <ul className="popup-content leaderboard-bottom">
+            {leaderBoard.map(({ name, id, time }) => (
+              <li className="popup-item" key={id}>
+                <span className="leaderboard-name">{name}</span>
+                <span className="popup-item-key">
+                  <span>{time}</span>
+                </span>
+              </li>
+            ))}
```
Comment (njm222) [inline src/ui/LeaderBoard.jsx:31]: maybe extract this out so Finished.jsx can use it as well? but everything else looks good!

**Review by njm222 — APPROVED** [review]

File: src/ui/LeaderBoard.jsx (hunk 2 above)
Comment (SaraVieira) [reply src/ui/LeaderBoard.jsx]: Done :)

**Comment by SaraVieira** [comment]
This also fixes the app crashing with no keys

**Pull request merged**